test/test_site/testIncludeMbd.mbd

```diff
@@ -0,0 +1 @@
+<p>Markbind supports .mbd files.</p>
```

Comment:
Might want to use some Markdown syntax in the test.

Also, please leave a newline at the end of the file so that we can avoid the `No newline at end of file` message (same for the other files).

Reply:
Sure 👍